i18n/test/babel-plugin.js

```diff
@@ -36,4 +42,47 @@ describe( 'babel-plugin', () => {
 			expect( isSameTranslation( a, b ) ).to.be.true();
 		} );
 	} );
+
+	describe( '.getTranslatorComment()', () => {
+		function getCommentFromString( string ) {
+			let comment;
+			traverse( transform( string ).ast, {
+				CallExpression( path ) {
+					comment = getTranslatorComment( path );
+				},
+			} );
+
+			return comment;
+		}
+
+		it( 'should not return translator comment on same line but after call expression', () => {
+			const comment = getCommentFromString( '__( \'Hello world\' ); // translators: Greeting' );
```

Comment:
So after scouring the [documentation](https://codex.wordpress.org/I18n_for_WordPress_Developers#Descriptions), [the code itself](http://develop.svn.wordpress.org/trunk/tools/i18n/extract.php), and [its tests](http://develop.svn.wordpress.org/trunk/tools/i18n/t/ExtractTest.php), I've come to the conclusions that:

- The comment must always occur before the translation call (see `$latest_comment` in `find_function_calls` token iteration)
- The PHP parsing is very non-particular about when that comment occurs. In fact, it looks like it could be many many lines prior the translate call, even if unrelated statements occur in-between. I'm not sure we care to preserve this behavior. It helps that the AST parsing is at least [very liberal with considering whitespace in `leadingComments`](http://astexplorer.net/#/gist/cb9162170ba191e1bc5de31e1dd7021e/96faface30e20f0e28c084c4271b403f19de8b88).
- In the PHP implementation the `translators:` is case-insensitive, which we should probably preserve here. I'll make that change shortly.